ONSdigital/srm-rh-ui
# 1082 rh UI implement pages and logic from old rh

# Motivation and Context

Now that we've got the base for the new rh-ui, we need to implement the rest of the logic and pages that RH-UI needs.
# What has changed



- Added all templates from old rh-ui
- Launch a survey
- Added welsh translations
- Translation logic simplified compared to old RH
- Error handling copied over from rh-ui 
# How to test?



- Bring up Docker-dev and stop the old rh-ui
- Run this branch against docker-dev
- Launch a survey
- Check all error routes (404, invalid uac length, invalid uac, no uac entered, Connection error to rh-service)
- Check all page links work as expected.
- Check all welsh translations work
- Run unit tests 
# Links



[Trello](https://trello.com/c/BkGDjNN4/)
